### PR TITLE
complete the task

### PR DESCRIPTION
**Pull request by barsuchellos**
image

### CODE REVIEW

**Review by poznianski — APPROVED** [review]
good

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,17 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```
Comment (silchyn) [inline src/herbivoresAndCarnivores.js:4]: What is the purpose of that 'health' parameter if you do not use it?

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,17 +1,43 @@
...
+  }
+
+  static died(animal) {
+    this.alive = this.alive.filter(el => el !== animal);
+  };
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
```
Comment (silchyn) [inline src/herbivoresAndCarnivores.js:17]: Unused parameter here also

File: src/herbivoresAndCarnivores.js [hunk 3]
```diff
@@ -1,17 +1,43 @@
...
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(el) {
```
Comment (silchyn) [inline src/herbivoresAndCarnivores.js:28]: What is el?

**Review by silchyn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**